File: Admin.NET/Admin.NET.Application/Entity/XzMonthfortunes.cs
// Admin.NET 项目的版权、商标、专利和其他相关权利均受相应法律法规的保护。使用本项目应遵守相关法律法规和许可证的要求。
//
// 本项目主要遵循 MIT 许可证和 Apache 许可证（版本 2.0）进行分发和使用。许可证位于源代码树根目录中的 LICENSE-MIT 和 LICENSE-APACHE 文件。
//
// 不得利用本项目从事危害国家安全、扰乱社会秩序、侵犯他人合法权益等法律法规禁止的活动！任何基于本项目二次开发而产生的一切法律纠纷和责任，我们不承担任何责任！

using Admin.NET.Core;
using SqlSugar;
namespace Admin.NET.Application.Entity;

/// <summary>
/// 星座月运势
/// </summary>
[Tenant("1300000000001")]
[SugarTable("xz_monthfortunes", "星座月运势")]
public partial class XzMonthfortunes : EntityBaseId
{
    /// <summary>
    /// 运势时间
    /// </summary>
    [SugarColumn(ColumnName = "ftime", ColumnDescription = "运势时间")]
    public virtual DateTime? ftime { get; set; }
    
    /// <summary>
    /// 星座类型
    /// </summary>
    [SugarColumn(ColumnName = "signtype", ColumnDescription = "星座类型")]
    public virtual int? signtype { get; set; }
    
    /// <summary>
    /// 综合分数
    /// </summary>
    [SugarColumn(ColumnName = "allscore", ColumnDescription = "综合分数")]
    public virtual int? allscore { get; set; }
    
    /// <summary>
    /// 爱情分数
    /// </summary>
    [SugarColumn(ColumnName = "lovescore", ColumnDescription = "爱情分数")]
    public virtual int? lovescore { get; set; }
    
    /// <summary>
    /// 健康分数
    /// </summary>
    [SugarColumn(ColumnName = "healthscore", ColumnDescription = "健康分数")]
    public virtual int? healthscore { get; set; }
    
    /// <summary>
    /// 财富分数
    /// </summary>
    [SugarColumn(ColumnName = "wealthscore", ColumnDescription = "财富分数")]
    public virtual int? wealthscore { get; set; }
    
    /// <summary>
    /// 幸运数字
    /// </summary>
    [SugarColumn(ColumnName = "luckynum", ColumnDescription = "幸运数字")]
    public virtual int? luckynum { get; set; }
    
    /// <summary>
    /// 幸运颜色
    /// </summary>
    [SugarColumn(ColumnName = "luckycolor", ColumnDescription = "幸运颜色", Length = 20)]
    public virtual string? luckycolor { get; set; }
    
    /// <summary>
    /// 幸运花
    /// </summary>
    [SugarColumn(ColumnName = "luckyflower", ColumnDescription = "幸运花", Length = 20)]
    public virtual string? luckyflower { get; set; }
    
    /// <summary>
    /// 幸运石
    /// </summary>
    [SugarColumn(ColumnName = "luckystone", ColumnDescription = "幸运石", Length = 20)]
    public virtual string? luckystone { get; set; }
    
    /// <summary>
    /// 解释
    /// </summary>
    [SugarColumn(ColumnName = "explaincontent", ColumnDescription = "解释", Length = 2000)]
    public virtual string? explaincontent { get; set; }
    
}

File: Web/src/api/xzmenu/xzMonthfortunes.ts
import {useBaseApi} from '/@/api/base';

// 星座月运势接口服务
export const useXzMonthfortunesApi = () => {
	const baseApi = useBaseApi("xzMonthfortunes");
	return {
		// 分页查询星座月运势
		page: baseApi.page,
		// 查看星座月运势详细
		detail: baseApi.detail,
		// 新增星座月运势
		add: baseApi.add,
		// 更新星座月运势
		update: baseApi.update,
		// 删除星座月运势
		delete: baseApi.delete,
		// 批量删除星座月运势
		batchDelete: baseApi.batchDelete,
		// 导出星座月运势数据
		exportData: baseApi.exportData,
		// 导入星座月运势数据
		importData: baseApi.importData,
		// 下载星座月运势数据导入模板
		downloadTemplate: baseApi.downloadTemplate,
	}
}

// 星座月运势实体
export interface XzMonthfortunes {
	// 主键Id
	id: number;
	// 运势时间
	ftime: string;
	// 星座类型
	signtype: number;
	// 综合分数
	allscore: number;
	// 爱情分数
	lovescore: number;
	// 健康分数
	healthscore: number;
	// 财富分数
	wealthscore: number;
	// 幸运数字
	luckynum: number;
	// 幸运颜色
	luckycolor: string;
	// 幸运花
	luckyflower: string;
	// 幸运石
	luckystone: string;
	// 解释
	explaincontent: string;
}